_posts/en/2020-05-20-Team-Topologies-Whitelisting-app.html

```diff
@@ -23,13 +23,25 @@
 
 The methodology outlined throughout this document is heavily borrowed from [Team Topologies](https://teamtopologies.com/?gclid=EAIaIQobChMI4N2R7fzM6AIV1f_jBx2o3ACIEAAYASAAEgL8DPD_BwE).
 
-![Team Topologies model]({{ site.baseurl }}/assets/images/TTBlog-Figure1.PNG)
-_Figure 1_
+![Figure 1: Team Topologies Model]({{ site.baseurl }}/assets/images/TTBlog-Figure1.PNG)
+
+Figure 1: Team Topologies model
+
+<details><summary>Text description of Figure 1</summary>
+This image shows a yellow box at the top representing the “Value Aligned Team”. This box is linked via XaaS to a blue box representing the “Platform Teams”.  Within the blue box, there are two platform teams that are also linked together via XaaS. In the case of the whitelisting application, the Value Aligned team is the IT Strategy team and the Platform Teams are DTS and SE.  Both of the “Platform teams” are linked internally via XaaS.  The SE platform team has one enabling contributor - Microsoft.</details>
+
+&nbsp;
 
 In a government context, the management model looked as follows
 
-![Government Team Topologies Model]({{ site.baseurl }}/assets/images/TTBlog-Figure2.PNG)
-_Figure 2_
+![Figure 2: Government Team Topologies Model]({{ site.baseurl }}/assets/images/TTBlog-Figure2.PNG)
+
+Figure 2: Government Team Topologies Model
+
```

Comment:
Oh, also, when injecting HTML directly in an MD file, you need to surround it with these tags so that our automated tests know to exclude the voluntary addition:

<!-- markdownlint-disable MD033-->
<!-- markdownlint-enable MD033-->

Reply:
10-4. On it. :)